geopandas/tests/test_dissolve.py

```diff
@@ -40,25 +42,39 @@ def setUp(self):
         self.mean['BoroCode'] = [4,1.5]
 
 
+    @unittest.skipIf(pd.__version__ < LooseVersion('0.16'), '...')
     def test_geom_dissolve(self):
         test = self.polydf.dissolve('manhattan_bronx')
         self.assertTrue(test.geometry.name == 'myshapes')
         self.assertTrue(test.geom_almost_equals(self.first).all())
 
+    @unittest.skipIf(pd.__version__ < LooseVersion('0.16'), '...')
     def test_first_dissolve(self):
         test = self.polydf.dissolve('manhattan_bronx')
-        test = test.drop('myshapes', axis=1)
-        first = self.first.drop('myshapes', axis=1)
-        assert_frame_equal(first, test)
+        assert_frame_equal(self.first, test, check_column_type=False)
 
+    @unittest.skipIf(pd.__version__ < LooseVersion('0.16'), '...')
     def test_mean_dissolve(self):
         test = self.polydf.dissolve('manhattan_bronx', aggfunc='mean')
-        test = test.drop('myshapes', axis=1)
-        mean = self.mean.drop('myshapes', axis=1)
-        assert_frame_equal(mean, test)
+        assert_frame_equal(self.mean, test, check_column_type=False)
 
         test = self.polydf.dissolve('manhattan_bronx', aggfunc=np.mean)
-        test = test.drop('myshapes', axis=1)
-        assert_frame_equal(mean, test)
+        assert_frame_equal(self.mean, test, check_column_type=False)
 
+    @unittest.skipIf(pd.__version__ < LooseVersion('0.16'), '...')
```

Comment:
Can you either drop the `'...'` or add some explanation in there? (I was just a bit brief in my example :-))